lib/install/process.go

```diff
@@ -51,25 +58,71 @@ func InitProcess(ctx context.Context, installerConfig Config, gravityConfig proc
 	return p, nil
 }
 
-// MakeProcessConfig creates a gravity process config from installer config
-func MakeProcessConfig(i Config) (*processconfig.Config, error) {
-	cfg, err := process.WizardProcessConfig(i.AdvertiseAddr, i.StateDir, i.WriteStateDir)
+// NewProcessConfig creates a gravity process config from installer config
+func NewProcessConfig(config ProcessConfig) (*processconfig.Config, error) {
+	assetsDir := filepath.Join(config.WriteStateDir, "assets")
+	if err := os.MkdirAll(assetsDir, defaults.SharedDirMask); err != nil {
+		return nil, trace.Wrap(trace.ConvertSystemError(err),
+			"failed to create directory for assets")
+	}
+	healthAddr, _ := teleutils.ParseAddr(fmt.Sprintf(":%v", defaults.WizardHealthPort))
+	adminRole, err := users.NewAdminRole()
 	if err != nil {
 		return nil, trace.Wrap(err)
 	}
-	cfg.ServiceUser = &i.ServiceUser
-	seedConfig, err := process.RemoteAccessConfig(i.StateDir)
+	wizardConfig := &processconfig.Config{
+		Mode:         constants.ComponentInstaller,
+		WebAssetsDir: assetsDir,
+		DataDir:      config.WriteStateDir,
+		Hostname:     config.AdvertiseAddr,
+		HealthAddr:   *healthAddr,
+		BackendType:  constants.BoltBackend,
+		Pack: processconfig.PackageServiceConfig{
+			ListenAddr: teleutils.NetAddr{
+				AddrNetwork: "tcp",
+				// Listen on all interfaces
```

Comment:
This was one of the requirements. I'm a little unsure about completely removing the ability to specify the advertise address for the wizard process (it is completely removed atm) and I might add this as a command line option.